setup.py

```diff
@@ -1,106 +1,70 @@
+import sys
+from distutils import log
 from pathlib import Path
-from shutil import move, rmtree
-from subprocess import check_call
-from tempfile import mkdtemp
 
 from setuptools import find_packages, setup
 from setuptools.command.build_py import build_py
 from setuptools.command.develop import develop
-from setuptools.command.sdist import sdist
 
-try:
-    # OSX Homebrew fix: https://stackoverflow.com/a/53190037/1067833
-    from sys import _base_executable as executable
-except ImportError:
-    from sys import executable
 
-_EGG_REQ_PATH = Path(__file__).parent.joinpath(
-    'py3_validate_email.egg-info', 'requires.txt')
-_REQ_PATH = Path(__file__).parent.joinpath('requirements.txt')
-
-with open(_REQ_PATH if _REQ_PATH.exists() else _EGG_REQ_PATH) as fd:
-    _req_content = fd.readlines()
-_DEPENDENCIES = [x.strip() for x in _req_content if x.strip()]
-
-with open(Path(__file__).parent.joinpath('README.rst')) as fd:
-    _LONG_DESC = fd.read()
-
-
-def run_initial_updater():
+def run_initial_updater(path):
     'Download an initial blacklist.txt on install time.'
-    # Install dependencies so the initial update can run
-    check_call([executable, '-m', 'pip', 'install'] + _DEPENDENCIES)
-    # The updater will walk code stack frames and see if this
-    # variable is set in locals() to determine if it is run from the
-    # setup, in which case it won't autoupdate.
-    _IS_VALIDATEEMAIL_SETUP = True
-    from validate_email.updater import BlacklistUpdater, LIB_PATH_DEFAULT
-    LIB_PATH_DEFAULT.mkdir(exist_ok=True)
-    blacklist_updater = BlacklistUpdater()
-    blacklist_updater._is_install_time = _IS_VALIDATEEMAIL_SETUP
-    blacklist_updater.process(force=True)
+    # Only import the updater module to avoid requiring all the dependencies
+    # and auto-running the updater.
+    sys.path.append(str(path.joinpath('validate_email')))
+    sys.dont_write_bytecode = True
```

Comment:
Please store and restore the original value of `dont_write_bytecode`, in order to not interfere with other installers.

Reply:
Okay.